cognite/client/_api_client.py

```diff
@@ -261,9 +261,7 @@ def _is_retryable(self, method: str, path: str) -> bool:
 
         if method in ["GET", "PUT", "PATCH"]:
             return True
-        if method == "POST" and self._url_is_retryable(path):
-            return True
-        return False
+        return bool(method == "POST" and self._url_is_retryable(path))
```

Comment:
We can drop the `bool`. We can also add in the first `if` check:
```py
return method in ["GET", "PUT", "PATCH"] or method == "POST" and self._url_is_retryable(path)
```
No, you don't need parentheses as `and` binds stronger than `or` 🚀

Reply:
Love it! The AND operator always ought to have priority when no bracketing, since as soon as one is not true, you can short-circuit that check and move along! 🚀 ⚡